tests/core/swapfiles/task.yaml

```diff
@@ -5,10 +5,11 @@ execute: |
   snap set system swap.size=200M
 
   echo "Check that the swap file was setup"
-  retry -n 60 --wait 1 bash -c "cat /proc/swaps | MATCH '\s+file\s+204796'"
+  # use swapon as it rounds up the numbers nicely
+  retry -n 60 --wait 1 bash -c "swapon --show | MATCH '\s+file\s+200M\s'"
```

Comment:
is this related to ubuntu-image ?

Reply:
Yes, for some reason there were byte size changes here (204796 -> 204784, 3 pages?). I suspect this can be caused by a different fs configuration, as looking at /usr/bin/mkswapfile the could should do the right and create 200MB sized file using fallocate and then runs mkswap on that. OTOH swapon --show correctly reports 200M size in both cases.

Reply:
hmm interesting, but sure makes sense